crates/sncast/tests/integration/lib_tests.rs

```diff
@@ -136,3 +137,10 @@ async fn test_get_account_failed_to_convert_field_elements() {
         .to_string()
         .contains("Failed to convert account address = address to FieldElement"));
 }
+
+#[tokio::test]
+async fn test_supported_rpc_version_matches_devnet_version() {
+    let provider = create_test_provider();
+    let devnet_spec_version = get_and_parse_spec_version(&provider).await.unwrap();
+    assert!(is_supported_version(&devnet_spec_version));
+}
```

Comment:
It doesn't really test anything from sncast, it should be a test for `shared` crate tbh

Reply:
I guess the idea was that this is the only check we can do in sncast (as we only use devnet for tests), so if we move the function to shared crate - I agree.

Reply:
The main rationale behind this decision was to avoid setting up devnet in the shared crate solely for this test. Although we could potentially run a similar test using a real node, it would currently require hardcoding node URLs in an additional location, so it's rather better to wait until #1690. 

For now we can either keep the test here or move it somewhere to the forge tests (given that we already have hardcoded urls there). Once we have envs in place we can add this test in the shared crate. Wdyt?

Reply:
Tbh I don't see an issue with hardcoing it in one more place and then using secrets when #1690 when it's introduced, it's easier this way then to remember to move the test imo

Reply:
> Tbh I don't see an issue with hardcoing it in one more place

Noo, let's not do this, please :( I think the best at the moment would be to move all those tests altogether to forge, and deal with extracting them to shared crate when the variable is taken out to env. I would much prefer this to leaving this test (with devnet) here.

Reply:
Up to u, but please make sure this test is moved to the shared crate after #1690

Reply:
yup, just as I said :+1:

Reply:
As discussed offline, we've agreed to leave this test as is for now. I've put a TODO note for it, and will make sure that we move it to the shared crate as soon as #1690 is done.